### ISSUE TITLE
Sur la page de profil et d'autres pages c'est pas tout a fait aligné au pixel perfect

### ISSUE THREAD

**Issue by firm1** [zestedesavoir/zds-site#4561]
Navigateur : Firefox 52
OS : Windows

Url de la page:

### Screen de la version de production

![pixel-perfect-yes](https://user-images.githubusercontent.com/6066015/29524403-04d63ef0-8690-11e7-9366-5f185a91ce90.png)

### Screen de la version de beta

![pixel-perfect](https://user-images.githubusercontent.com/6066015/29524392-fe5d03c4-868f-11e7-987a-384fae832e32.png)

On voit que les choses ne sont pas alignés, je trouve ça moins esthétique que ce qu'on avait jusqu'ici en production.

Le même problème d'alignement se retrouve sur certaines pages comme celles-ci : 

- La pages "mes tutoriels"
- La page "mes billets"
- La page "mes articles"
- La page des [demandes d'aide](https://beta.zestedesavoir.com/contenus/aides/)
- La page [tous les billets](https://beta.zestedesavoir.com/billets/)
- Et j'en ai peut-être raté d'autres

**Comment by motet-a** [comment]
Oui, c’est moi qui ai indroduit ça un peu volontairement, c’est en attente de reprendre les choses plus correctement.

Bon, ça peut sembler pas trop justifié, mais dans l’effort de corriger des problèmes importants comme le footer qui ne se mettait pas toujours en bas de la page et d’autres moins importants comme l’alignement du menu (qui sera d’ailleurs parfaitement centré quand j’aurais le temps de modifier les quelques lignes de CSS qui manquent), j’ai remplacé pas mal de tailles en pourcentages par du `display: flex;` et autres. Du coup, le nouveau header n’est pas aligné avec le reste de la page qui est encore dimensioné avec plein de pourcentages.

**Comment by firm1** [comment]
Tant que le problème est connu, j'imagine que ça va finir par être corrigé. Mais ça ne me semble pas urgentissime, je le note juste pour pas qu'on le jette aux oubliettes.
